src/pages/index.tsx

```diff
@@ -7,11 +7,32 @@ import { DashboardPage } from './DashboardPage';
 import { EmbalmPage } from './EmbalmPage';
 import { FundBundlrPage } from './FundBundlrPage';
 import { TempResurrectionPage } from './TempResurrectionPage';
+import { useNetworkConfig } from 'lib/config';
+import { networkConfigs } from 'lib/config/networkConfig';
+import { useAccount } from 'wagmi';
+
+export enum RouteKey {
+  EMBALM_PAGE,
+  DASHBOARD_PAGE,
+  DASHBOARD_DETAIL,
+  ARCHEOLOGIST_PAGE,
+  BUNDLER_PAGE,
+  TEMP_RESURRECTION_PAGE,
+}
+
+export const RoutesPathMap: { [key: number]: string } = {
```

Comment:
Created this `map` for the route paths. I'm not 100% sure of the name and would take any suggestion. Or even a different way to store those? Maybe just a helper function.

Reply:
I think this is the logical place for this. It's an improvement but I'm afraid people will forget about it and just use the old way. Creating a custom `navigate` function that only accepts the `RouteKey` might help people use it properly, but they still might use the old function.